db/queries/choices.js

```diff
@@ -22,20 +32,21 @@ const getChoices = function(pollId) {
 
 const getChoicesandscore = function(pollId) {
   return db.query(`
-  SELECT choices.id AS choice_id, value, sum(6 - votes.ranking) AS score FROM votes
-  RIGHT JOIN choices on choices.id = votes.choice_id
-  WHERE choices.poll_id = $1
-  GROUP BY choices.id, value
-  ORDER BY choice_id;
-  `, [pollId])
-    .then((result) => {
-      return result.rows;
-    })
-    .catch((err) => {
-      console.log(err.message);
-    });
+  SELECT choices.id,value, sum(6-votes.ranking) as score FROM choices
```

Comment:
please use the master version of the query, it has been refactored to fix the zero vote bug.